add achieved target amount projects 알림 구성

nowgnas opened the pull request:
## pull request 설명
- 프로젝트 목표 금액 달성 정보 sqs로 전달하기 구현 

### 변경 사항

- [ ] fix bug
- [ ] add comment
- [x] add new feature
- [ ] update documents
- [ ] Improvement(refactoring and improving code)
- [ ] etc

nowgnas committed:
add: add achieved target amount projects 알림 구성

nowgnas committed:
fix: 코드 구성 변경 close #6

- 반환이 필요하지 않는 부분 변경

nowgnas committed:
Merge pull request #8 from nowgnas/feature/funding-target-amount-notification

 add achieved target amount projects 알림 구성

CokeLee777 commented on src/main/java/com/lotdiz/projectservice/mapper/ProjectMapper.java:
```diff
@@ -0,0 +1,23 @@
+  ProjectInformationForAchievedTargetAmountRequestDto
+      getProjectInformationForAchievedTargetAmountRequestDto(Project project);
+
+  @IterableMapping(qualifiedByName = "PIFATARD")
```
이건 뭔가요? 첨보네요

nowgnas replied:
```
@Named("mapping-name")
ProjectDto toDto(Project project);

@IterableMapping(qualifiedByName = "mapping-name")
List<ProjectDto> toList(List<Project> projects);
```
이렇게 리스트 형태를 한번에 원하는 dto 객체 리스트로 매핑 해주는 동작입니다~

CokeLee777 reviewed (commented):
확인했습니다~ 위에 뭔지만 알려주세요~

nowgnas commented:
추가 수정 사항 반영 후 머지 하겠습니다

nowgnas committed:
add: add achieved target amount projects notification 을 위한 로직 구현 close #6

- event bridge -> sqs <- 메세지 폴링
- 프로젝트 조회 후 funding service에 데이터 요청
- notification service에 저장할 알림 정보를 위한 sqs publish 구현

nowgnas committed:
Merge pull request #11 from nowgnas/feature/funding-target-amount-notification

add: add achieved target amount projects notification 을 위한 로직 구현 close #6

nowgnas committed:
Merge branch 'develop' of https://github.com/nowgnas/lotdiz-project-service into feature/funding-target-amount-notification

nowgnas committed:
Merge pull request #12 from nowgnas/feature/funding-target-amount-notification

Feature/funding target amount notification merge branch

CokeLee777 reviewed (approved):
good

nowgnas committed:
fix: delete postman mock url
